box-content-sdk/src/main/java/com/box/androidsdk/content/requests/BoxRequestsFile.java

```diff
@@ -1086,5 +1088,22 @@ protected void onSendCompleted(BoxResponse<BoxVoid> response) throws BoxExceptio
             super.onSendCompleted(response);
             super.handleUpdateCache(response);
         }
+
+        /**
```

Comment:
This might be too confusing for external users. Where is this used from - can we have it somewhere in the app for the time being?